### DISCUSSION TITLE
Index status modification

### DISCUSSION THREAD

**Discussion by pzadsl** [JanusGraph/janusgraph#2553] [Q&A]
**Describe the feature:**
[Please add a description for your feature.]

**Describe a specific use case for the feature:**
[If possible add a description of the specific use case.]

The same library, labels, and indexes are created in batches. The second label and index can be created successfully, but the status of the second label index cannot be modified, indicating a null pointer exception. 

Dispatch an uncaught exception to the handler. This method is
       intended to be called only by the JVM.
![j1](https://user-images.githubusercontent.com/16890385/112777783-a3e1f500-9075-11eb-81fa-fab34d304cee.png)
![j2](https://user-images.githubusercontent.com/16890385/112777798-ac3a3000-9075-11eb-9f44-0d923ae68804.png)

**Comment by storyteller28th** [comment]
you should check your index status!!!
